Update_CRO_integreatly-operator.md

```diff
@@ -1,4 +1,4 @@
-# Adding Latest CRO to RHMI
+# Adding Latest CRO to RHOAM
```

Comment:
Could we make this doc more generic for any use of the operator. or if not applicable here, move it to the integreatly-operator repo

Reply:
https://github.com/RHCloudServices/integreatly-help/pull/827/commits/843620581279678b3a888889418e540da2e340be